api/tests/Unit/Service/PreRegistrationVerificationServiceTest.php

```diff
@@ -26,46 +26,54 @@ public function setUp(): void
         $crLayHasPC = m::mock('\App\Entity\PreRegistration')
```

Comment:
I remember updating this test when we moved to pre-registration wording, it's horrendous!